tools/sgx-lkl-java

```diff
@@ -11,7 +11,8 @@ else
 fi
 
 SGX_LKL_OPTIONS="\
-  SGXLKL_MMAP_FILES=${SGXLKL_MMAP_FILES:-Shared}"
+  SGXLKL_MMAP_FILES=${SGXLKL_MMAP_FILES:-Shared} \
+  SGXLKL_HOST_IMPORT_ENV=JRE_LIB_PATH,DEFAULT_JRE_ARGS,LD_LIBRARY_PATH"
```

Comment:
DEFAULT_JRE_ARGS is not an env var. JRE_LIB_PATH is only used locally. The only thing needed is LD_LIBRARY_PATH.